### DISCUSSION TITLE
How do you use this?

### DISCUSSION THREAD

**Discussion by srid** [BamButz/msbuild-tailwindcss#6] [General]
[I'm new to dotnet ecosystem]

So I've got a Bolero web project in VS 2019, and added this package as a NuGet dependency (on top of [this project](https://github.com/srid/Baseline)). But how do I actually use it? 

I have node & npm installed (native Windows version). There is no package.json in my project, but I figured this build action will create for me? The CSS file is [located here](https://github.com/srid/Baseline/blob/master/src/Baseline.Client/wwwroot/css/index.css).

**Comment by srid** [comment]
Ah, so I have to do this (set a "build action" for the .css file in question):

![image](https://user-images.githubusercontent.com/3998/112050456-41de3880-8b27-11eb-957a-4fb8b8c7d7fe.png)

And the reference the generated CSS file. Great.

**Reply by BamButz** [reply]
Yea, that's how it should be used.
So, then I assume you have answered your question yourself. 🙂
